standard/unsafe-code.md

```diff
@@ -958,17 +958,18 @@ A fixed-size buffer can be referenced in an expression using a *simple_name* ([
 
 When a fixed-size buffer member is referenced as a simple name, the effect is the same as a member access of the form `this.I`, where `I` is the fixed-size buffer member.
 
-In a member access of the form `E.I`, if `E` is of a struct type and a member lookup of `I` in that struct type identifies a fixed-size member, then `E.I` is evaluated and classified as follows:
+In a member access of the form `E.I` where `E.` may be the implicit `this.`, if `E` is of a struct type and a member lookup of `I` in that struct type identifies a fixed-size member, then `E.I` is evaluated and classified as follows:
 
 - If the expression `E.I` does not occur in an unsafe context, a compile-time error occurs.
 - If `E` is classified as a value, a compile-time error occurs.
-- Otherwise, if `E` is a moveable variable ([§22.4](unsafe-code.md#224-fixed-and-moveable-variables)) and the expression `E.I` is not a *fixed_pointer_initializer* ([§22.7](unsafe-code.md#227-the-fixed-statement)), a compile-time error occurs.
+- Otherwise, if `E` is a moveable variable (§22.4) then:
+  - If the expression `E.I` is a *fixed_pointer_initializer* (§22.7), then the result of the expression is a pointer to the first element of the fixed size buffer member `I` in `E`.
+  - Otherwise if the expression `E.I` is a *primary_no_array_creation_expression* within an *element_access* expression of the form `E.I[J]` then then the result of the expression is the result of the *primary_no_array_creation_expression* (§11.7.10.1).
```

Comment:
Maybe a bit more expansion of my "yada, yada" as there is nothing now about being evaluated like a *pointer_element_access* – or was this equivalence more  illustrative?

If not then maybe:

> - Otherwise if the expression `E.I` is a *primary_no_array_creation_expression* (§11.7.10.1) within an *element_access* (§11.7.10) of the form `E.I[J]`, then the result of `E.I` is a pointer, `P`, to the first element of the fixed size buffer member `I` in `E`, and the enclosing *element_access* is then evaluated as the *pointer_element_access* (§22.6.4) `P[J]`.

Reply:
Yes! It is a *primary_no_array_creation_expression* that is a *pointer_element_access*.